### PR TITLE
some spelling and grammar, and echo $?

### PR DESCRIPTION
**Pull request by elspru**
Just a few fixes to the parts I read,
good stuff otherwise! Exciting bringing people into the world of the command line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,17 +58,19 @@ Here's a breakdown of what you just did:
 ### Creating a file (touch) :point_down:
 Let's make a file. In a GUI program with a nice user interface, we might right click in the file explorer, and click 'New File', but we can't do that in the terminal.
 
-Instead we will use a command called `touch`. This will create an empty file for us. Let's type `touch myFile.txt` into the terminal.
+Instead, we will use a command called `touch`. This will create an empty file for us. Let's type `touch myFile.txt` into the terminal.
 
-Once you've done that, you might notice you don't get any feedback, there's nothing that says "File created succesfully" or anything, you just see a new prompt to enter a new command. This is normal, in Unix there is a convention that **if a program doesn't output anything, it executed succesfuly**.
+Once you've done that, you might notice you don't get any feedback, there's nothing that says "File created successfully" or anything, you just see a new prompt to enter a new command. This is normal, in Unix there is a convention that **if a program doesn't output anything, it executed successfully**.
 
+You can also check what the return code was with `echo $?`
```
Comment (zvory) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -58,17 +58,19 @@ Here's a breakdown of what you just did:
...
 Let's make a file. In a GUI program with a nice user interface, we might right click in the file explorer, and click 'New File', but we can't do that in the terminal.
 
-Instead we will use a command called `touch`. This will create an empty file for us. Let's type `touch myFile.txt` into the terminal.
+Instead, we will use a command called `touch`. This will create an empty file for us. Let's type `touch myFile.txt` into the terminal.
 
-Once you've done that, you might notice you don't get any feedback, there's nothing that says "File created succesfully" or anything, you just see a new prompt to enter a new command. This is normal, in Unix there is a convention that **if a program doesn't output anything, it executed succesfuly**.
+Once you've done that, you might notice you don't get any feedback, there's nothing that says "File created successfully" or anything, you just see a new prompt to enter a new command. This is normal, in Unix there is a convention that **if a program doesn't output anything, it executed successfully**.
 
+You can also check what the return code was with `echo $?` 
+if the answer is 0 then it was a successful exit.
```
Comment (zvory) [inline README.md]:
```suggestion
```
This is not something I'd put in the 101

**Comment by elspru** [comment]
i made the changes you wanted but don't know how  to make a new pull request, github makes it very difficult.

**Comment by zvory** [comment]
no need to create a new PR, this PR has been updated to include my suggestions

**Pull request merged**